pkg/kustomize/kustomize_cli.go

```diff
@@ -0,0 +1,42 @@
+package kustomize
+
+import (
+	"strings"
+
+	"github.com/jenkins-x/jx/pkg/util"
+)
+
+// KustomizeCLI implements common kustomize actions based on kustomize CLI
+type KustomizeCLI struct {
+	Runner util.Commander
+}
+
+// NewKustomizeCLI creates a new KustomizeCLI instance configured to use the provided helm CLI in
```

Comment:
I guess the Godocs are still off here, right? - "configured to use the provided helm CLI"